paparazzi/src/main/java/app/cash/paparazzi/Environment.kt

```diff
@@ -29,27 +27,20 @@ data class Environment(
   val assetsDir = "$appTestDir/src/main/assets/"
 }
 
-fun detectEnvironment(): Environment {
+fun detectEnvironment(paparazziResourcesDetailsFile: String = System.getProperty("PAPARAZZI_RESOURCES_DETAILS_FILE_KEY")): Environment {
```

Comment:
i like the system property.  does it need to be passed as an argument, vs looking it up similar to the properties below?  if not necessary, let's remove the argument.

Reply:
I provided a default value to the argument, but I'm not sure it is possible to detect where it is located because of variant paths, build-system, and plugin version.

Reply:
yea, using the system property is fine.  i'm suggesting to move it out of the argument list, and have it similar to the userDir system property below.